src/sound_openal.cpp

```diff
@@ -575,6 +599,47 @@ class OpenALSoundManager: public ISoundManager
 		maintain();
 		deleteSound(sound);
 	}
+
+	void fadeSound(int soundid, float step, float gain)
+	{
+		float cGain = getSoundGain(soundid);
+		FadeState f = FadeState(step,cGain,gain);
+		m_sounds_fading[soundid] = f;
+	}
+
+	void doFades(float dtime)
+	{
+		static float fadeDelay = 0;
+		fadeDelay += dtime;
+		if ( fadeDelay > 0.2 ){
```

Comment:
Please make sure that all your if-checks are in the following format: `if (number > number) {`. No matter if the code around is formatted wrongly.